src/main/java/Duke.java

```diff
@@ -1,10 +1,168 @@
+import java.util.Scanner;
+
 public class Duke {
+    /* immutable Strings below
+     *
+     *
+     */
+    public static final String GREETING = " Hello! I'm Duke! What can I do for you?";
+
+    public static final String INVALID = " Invalid command!";
+    public static final String GOODBYE = " Bye. Hope to see you again soon!";
+    public static final String LINE_DIVIDER = "____________________________________________________________";
+    public static final int MAX_TASKS = 101;
+
+    /* Storage and variables below
+     *
+     *
+     */
+    private static int indexBaseOne;
+    private static Task[] tasks;
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    private static String[] commandWords;
+
+    private static String commandWord;
+
+    private static String commandParameters;
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        greet();
+        initTasks();
+
+        do {
+            decodeUserInputs();
+
+            resolveUserInputs();
+
+            lineBreak();
+
+
+        } while (true);
+    }
+
+    private static void resolveUserInputs() {
+        if (commandWord.equals("bye")) {
+            goodbye();
+        }
+
+        switch (commandWord) {
+            case "list":
+                printTaskList();
+
+                break;
+            case "mark": {
+                markTaskDone();
+
+                break;
+            }
+            case "unmark": {
+                MarkTaskNotDone();
+                break;
+            }
+            case "todo":
+                addToDo();
+
+                break;
+            case "deadline":
+                addDeadline();
+
+                break;
+            case "event":
+                addEvent();
+
+                break;
+            default:
+                System.out.println(INVALID);
+                System.out.println( "input: " + commandWord + "," + commandParameters);
+                break;
+        }
+    }
```

Comment:
Do note that for the class, the coding standards states there should not be indentations for case statements.